content/02.body.md

```diff
@@ -0,0 +1,31 @@
+## Introduction
+
+### Node degree bias
+
+### Unsupervised link prediction
```

Comment:
Why just unsupervised? Also I think we should pick a single term for edge/relationship/link. I'd vote for "edge". Then in text you can say something like `edge prediction (frequently also referred to as "link prediction")`, such that you can get the SEO.